### PR TITLE
black and lint the code

### PR DESCRIPTION
**Pull request by lexming**

### CODE REVIEW

File: lib/vsc/filesystem/oceanstor.py [hunk 1]
```diff
@@ -310,7 +310,7 @@ def __init__(self, url, account, username, password):
 
         self.vsc = VSC()
         self.vscstorage = VscStorage()
-        self.host_institute = vsc.options.options.host_institute
+        self.host_institute = self.vsc.options.options.host_institute
```
Comment (wpoely86) [inline lib/vsc/filesystem/oceanstor.py:313]: it's not enough. Add a call to `self.vsc.read_cfg()`

Comment (lexming) [reply lib/vsc/filesystem/oceanstor.py:313]: not needed, that's done by default

Comment (wpoely86) [reply lib/vsc/filesystem/oceanstor.py:313]: yes but `VscStorage()` explicitly calls it with `False` as arg...

Comment (lexming) [reply lib/vsc/filesystem/oceanstor.py:313]: `VscStorage` is not playing any role here...
```
>>> from vsc.config.base import VSC
>>> vsc = VSC()
>>> vscsto = VscStorage()
>>> vsc.options.options.host_institute
'brussel'
```

Comment (wpoely86) [reply lib/vsc/filesystem/oceanstor.py:313]: it's a singleton class.

Trust me, I've tried this already on storctrl. I won't work :wink:

Comment (lexming) [reply lib/vsc/filesystem/oceanstor.py:313]: ok, I see what you mean, this is a Python 2 thing. Fixed in https://github.com/vub-hpc/vsc-filesystem-oceanstor/pull/23/commits/7c39a993976aefb3fea207da35dce9beb9433964
```
Python 2.7.5 (default, Jun 28 2022, 15:30:04)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import config
>>> import oceanstor
>>> O = oceanstor.OceanStorOperations(config.OCEANSTOR_URL, config.OCEANSTOR_ACCOUNT, config.OCEANSTOR_USER, config.OCEANSTOR_PWD)
>>> O.host_institute
'brussel'
```

**Review by wpoely86 — APPROVED** [review]

**Pull request merged**